src/ngraph/runtime/cpu/cpu_cse.cpp

```diff
@@ -29,7 +29,25 @@ using namespace std;
 
 static bool cse_convertlayout(std::shared_ptr<Node> a, std::shared_ptr<Node> b)
 {
-    return false;
+    NGRAPH_DEBUG << "In cse_convertlayout for " << a->get_name() << " and " << b->get_name();
+
+    auto ar_a = std::static_pointer_cast<runtime::cpu::op::ConvertLayout>(a);
+    auto ar_b = std::static_pointer_cast<runtime::cpu::op::ConvertLayout>(b);
+
+    auto get_tvl = [](std::shared_ptr<Node> node) {
```

Comment:
Since we don't have tensor views any more, we should call these something other than tv. Tensor?

Reply:
sure make sense.

Reply:
@diyessi done.